tests/yaml/issue-400.yaml

```diff
@@ -100,13 +168,41 @@ table: |
   begcaps 456
   endcaps 5
 
+# Forward tests (see above)
+tests:
+  # endcaps cancels a block in capitals
+  - - "ABCabc"
+    - ",abc.abc"
+  # Whitespace does not cancel a block in capitals
+  - - "ABC ABC"
+    - ",abc abc."
+  # Non-letter cancels a block in capitals
+  - - "ABC%abc"
+    - ",abc%abc"
+    - {xfail: true} # FIXME: undefined behavior?
+    # Actual: ',abc%.abc'
+    # Buevest:
+    # The actual result seems more logical to me.
+    # Capsphrase should be able to handle phrases like:
+    # HE, THE MAN, SAID IT ACTUALLY WORKED.
+    # Perhaps, endcapsphrase should be placed before %,
+    # but that is not the topic of this issue.
+
+  # Number cancels a block in capitals
+  - - "ABC123abc"
+    - ",abc#abc;abc"
+    - {xfail: true}  # FIXME: undefined behavior?
+    # Actual: ',abc#abc.;abc'
```

Comment:
Do you really think this result is logical? Some other possibilities are:
- ,abc#abc;.abc
- ,abc.#abc;abc
- ,abc#abc.abc

Does it say something about this case in the UEB manual?

Note that there are languages (notably Dutch) that have only one "cancel sign" that cancels numbers as well as capitals and emphasis.